src/bp/ui-studio/src/web/views/FlowBuilder/inspector/index.tsx

```diff
@@ -21,71 +23,105 @@ import StandardNode from '../nodeProps/StandardNode'
 import style from './style.scss'
 
 interface Props {
-  currentFlowNode: any
+  buffer: any
   closeFlowNodeProps: any
-  show: any
+  copyFlowNode: any
+  copyFlowNodeElement: any
+  currentFlow: any
+  currentFlowNode: any
+  flows: any
   history: any
+  onDeleteSelectedElements: () => void
+  pasteFlowNode: any
+  pasteFlowNodeElement: any
   readOnly: any
-  updateFlowNode: any
   refreshFlowsLinks: any
-  flows: any
-  currentFlow: any
   requestEditSkill: any
-  copyFlowNodeElement: any
-  pasteFlowNodeElement: any
-  buffer: any
+  show: any
   updateFlow: any
+  updateFlowNode: any
   user: any
 }
```

Comment:
Can you separate props with the new typings which get types from the mapStateToProps directly ?